src/sentry/event_manager.py

```diff
@@ -871,6 +872,8 @@ def _save_aggregate(self, event, hashes, release, **kwargs):
                     **kwargs
                 ), True
 
+            metrics_incr('group.created', skip_internal=True, tags={'platform': event.platform})
```

Comment:
So it's worth noting that platform may be `None`. Not sure if that'll be problematic if that's passed through, but may want to handle it as like, `event.platform or 'unknown'` ?